portal/src/main/scala/cromwell/pipeline/datastorage/dto/User.scala

```diff
@@ -27,3 +31,16 @@ object UserId {
 }
 
 final case class ProfilePicture(value: Array[Byte]) extends MappedTo[Array[Byte]]
+
+object ProfilePicture {
+  implicit lazy val profilePictureFormat: OFormat[ProfilePicture] = OFormat(profilePictureReads, profilePictureWrites)
+
+  lazy val profilePictureWrites = new OWrites[ProfilePicture] {
+    def writes(profilePicture: ProfilePicture) = Json.obj(
+      "value" -> None
+    )
+  }
+
+  lazy val profilePictureReads: Reads[ProfilePicture] =
+    (JsPath \ "value").read(Reads.of[Array[Byte]]).map(ProfilePicture.apply _)
```

Comment:
I've just noticed that this is the wrong place to hide `ProfilePicture` content. The content of the profile picture is an array, it is strange to replace it with `None`. You must provide custom `Reads` and `Writes` for a `User` class in order to always write `profilePicture` as `None`.
Also, why do you need to do it right now? :)

Reply:
done, thanks for explanation